packages/dds/shared-object-base/src/serializer.ts

```diff
@@ -29,7 +29,9 @@ export const isSerializedHandle = (value: any): value is ISerializedHandle =>
 	value?.type === "__fluid_handle__";
 
 /**
- * @alpha
+ * @privateRemarks
+ * used by fluidframework/map public SharedMap
+ * @public
  */
 export interface IFluidSerializer {
```

Comment:
If this type is to be elevated to public, we should replace the use of `any`. `unknown` would be a step in the right direction, but not great. Use of Serializable and companion "Deserialized" types preferred.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6597